app/models/journeys/configuration.rb

```diff
@@ -0,0 +1,29 @@
+# Journey-specific configuration, managed through the service operator's admin
+# interface.
+#
+# Things that are currently configurable:
+#
+# * open_for_submissions: defines whether the journey is currently accepting
+#   claims or not
+# * availability_message: an optional message that is shown to users when the
+#   journey is closed for submissions
+# * current_academic_year: the academic year the service is currently accepting
+#   claims for.
+module Journeys
+  class Configuration < ApplicationRecord
+    self.table_name = "journey_configurations"
+
+    # Use AcademicYear as custom ActiveRecord attribute type
+    attribute :current_academic_year, AcademicYear::Type.new
+
+    validates :current_academic_year_before_type_cast, format: {with: AcademicYear::ACADEMIC_YEAR_REGEXP}
+
+    def additional_payments?
+      journey == Journeys::AdditionalPaymentsForTeaching
+    end
```

Comment:
Good idea, though so far only `additional_payments?` is used anywhere (in the admin manbage services section). Perhaps when we add the new policies this will be useful.

Reply:
Definitely, we'll probably end up doing it at some point anyway, I think it's good to keep as much abstraction as possible with this new model